apps/meteor/server/models/raw/LivechatVisitors.ts

```diff
@@ -37,7 +37,6 @@ export class LivechatVisitorsRaw extends BaseRaw<ILivechatVisitor> implements IL
 			{ key: { username: 1 } },
 			{ key: { 'contactMananger.username': 1 }, sparse: true },
 			{ key: { 'livechatData.$**': 1 } },
-			{ key: { activity: 1 }, partialFilterExpression: { activity: { $exists: true } } },
```

Comment:
Can you leave this commented with a note that we have to remove it? otherwise it may end up being forgotten for next major

Reply:
By commenting the index, aren't we already removing it?

Reply:
> By commenting the index, aren't we already removing it?

We're only preventing thisi ndex from being created in new workspaces, I believe
We'd still need a migration to drop it in older workspaces that had this index created